LaunchDarkly/LaunchDarkly/Networking/DarklyService.swift

```diff
@@ -2,7 +2,12 @@ import Foundation
 import LDSwiftEventSource
 
 #if os(Linux) || os(Windows)
-import FoundationNetworking
+import class FoundationNetworking.URLResponse
+import class FoundationNetworking.HTTPURLResponse
+import struct FoundationNetworking.URLRequest
+
+import class FoundationNetworking.URLSessionConfiguration
+import AnyURLSession
```

Comment:
Have you considered using an `Exports.swift` file to `AnyURLSession` for these other imports? That way they don't have to be repeated here.